ops/model.py

```diff
@@ -2012,7 +2012,7 @@ def push(self,
              user_id: Optional[int] = None,
              user: Optional[str] = None,
              group_id: Optional[int] = None,
-             group: Optional[str] = None):
+             group: Optional[str] = None) -> None:
```

Comment:
Pyright requires the `ContainerFilesystem` Protocol to have completed typing (i.e. with return types). So to make checking the consistency between `Container` and `ContainerFilesystem` I added return types to the `Container` class as well.

Reply:
Not sure why this change was required, but an explicit `-> None` does seem to be what's recommended by [PEP 484](https://peps.python.org/pep-0484/). Part of the reason is that type checkers don't check functions which don't have any type annotations, and if you leave `-> None` off, for a function with no arguments like `def foo(): ...` the type checkers wouldn't check it, but they would check `def foo() -> None: ...` because it has type annotations.